campaign_info.toml

```
@@ -6,9 +6,9 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C23_WMDE_Desktop_00"
-description = "3 step form vs 4 step form"
-campaign_tracking = "00-ba-230501"
+campaign = "C23_WMDE_Desktop_DE_01"
+description = "Entire banner text change"
```

```suggestion
description = "Text change: Wikipedia's usefulness"
```
Using the wording from the ticket might help later finding that ticket or finding out  what the text change was